core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1170,16 +1170,6 @@ package object config {
       .toSequence
       .createWithDefault(Nil)
 
-  private[spark] val EXECUTOR_PLUGINS =
-    ConfigBuilder("spark.executor.plugins")
```

Comment:
https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/SparkConf.scala#L623-L624

This conf is removed, but it is still part of "spark.executor.plugins"?

Reply:
This was pointed out above https://github.com/apache/spark/pull/26390#discussion_r343960667. Probably we should fix it to make it clear.